examples/hello_world_component/Makefile

```diff
@@ -6,23 +6,21 @@
 .PHONY: all
 
 ifeq ($(BUILD_TYPE),Debug)
-  hpx_application = hpx_application_debug
-  hpx_component = hpx_component_debug
   debug_suffix = d
 else
-  hpx_application = hpx_application
-  hpx_component = hpx_component
   debug_suffix =
```

Comment:
These need to go back?

Reply:
Yep forgot it, I just changed the names to _release and _debug, it seems clearer to me

Reply:
Good idea!